src/world.js

```diff
@@ -276,6 +279,18 @@ $.extend( $.World.prototype, $.EventSource.prototype, /** @lends OpenSeadragon.W
     },
 
     /**
+     * As a performance optimization, setting this flag to false allows the bounds-change event handler
+     * on tiledImages to skip calls to _figureSizes. If a lot of images are going to be positioned in
```

Comment:
Suddenly I'm realizing the problem here is that it implies people should call _figureSizes after setting the auto refigure back on, and the purpose of the underscore in _figureSizes's name is to mark it as private. In other words, it should be essentially invisible in the API. 

So, one option would be to promote it to a public method (i.e. remove the underscore). I'm hesitant to do that, since it's not really useful outside the context of setAutoRefigureSizes. Another option would be for setAutoRefigureSizes to automatically call _figureSizes if `value` is true. As an added nicety, we could keep track of whether _delegatedFigureSizes got called since _autoRefigureSizes was set to false, and only _refigureSizes when it gets set back to true in that case. 

What do you think?